src/evaluators/bc/PHAL_DirichletField_Def.hpp

```diff
@@ -257,18 +289,10 @@ evaluateFields(typename Traits::EvalData dirichletWorkset) {
         int lunk = nsNodes[inode][this->offset];
         GO node_gid = nsNodesGIDs[inode];
         int lfield = fieldDofManager.getLocalDOF(field_node_indexer->getLocalElement(node_gid),fieldOffset);
-        for (int col=0; col<num_cols; ++col) {
+	      for (int col=0; col<num_cols; ++col) {
```

Comment:
Alignment looks off here.

Reply:
thanks, fixed this and the other indentation errors.